homeassistant/components/livisi/config_flow.py

```diff
@@ -39,19 +40,22 @@ async def async_step_user(
             await self._login(user_input)
         except livisi_errors.WrongCredentialException:
             errors["base"] = "wrong_password"
-        except livisi_errors.ShcUnreachableException:
+        except livisi_errors.ShcUnreachableException as exception:
             errors["base"] = "cannot_connect"
+            raise ConfigEntryNotReady from exception
```

Comment:
We don't want to raise `ConfigEntryNotReady` from the config flow. We're not setting up the config entry here. We're gathering information to create a config entry or update an existing config entry.

Reply:
I removed the changed from `config_flow`.